### DISCUSSION TITLE
Clearcoat material

### DISCUSSION THREAD

**Discussion by scerelli** [mozilla/hubs#3816] [Art Questions] [answered]
Hi guys do you know how to get clearcoat material working on Mozilla hubs? i read the khronos specification and seems like clearcoat is supported by gltf, but in spoke, if i validate the glb file i get unsupported extension: "KHR_materials_clearcoat". I tried to load the model via three.js and it also works fine.

Of course when i publish the scene the material it's the wrong one.

**Comment by cvanmeurs** [comment]
We don't currently support clearcoat.  A fairly comprehensive breakdown of the texture types we support can be found here:  https://www.khronos.org/blog/art-pipeline-for-gltf

**Comment by j-conrad** [comment, marked as answer]
Good news-- Hubs recently updated the version of threejs that gives it support for both Clearcoat and Transmission. (r133)
